.github/workflows/trustee-attester.yml

```diff
@@ -0,0 +1,99 @@
+name: trustee-attester basic build
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - 'attestation-agent/**'
+      - '.github/workflows/trustee-attester.yml'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+  pull_request:
+    paths:
+      - 'attestation-agent/**'
+      - '.github/workflows/trustee-attester.yml'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+  create:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  trustee_attester_ci:
+    if: github.event_name != 'push'
```

Comment:
why this limitation?

Reply:
Same as in aa_basic.yml

Reply:
I think that's a bug. let's remove it here.